applications/callflow/src/module/cf_conference.erl

```diff
@@ -30,6 +30,8 @@ handle(Data, Call) ->
            ,{<<"Play-Welcome">>, wh_json:is_true([<<"welcome_prompt">>, <<"play">>], Data, 'true')}
            ,{<<"Play-Welcome-Media">>, wh_json:get_ne_value([<<"welcome_prompt">>, <<"media_id">>], Data)}
            ,{<<"Conference-Doc">>, wh_json:get_value(<<"config">>, Data)}
+           ,{<<"Play-Exit-Tone">>, wh_json:get_ne_value([<<"play_exit_tone">>], Data)}
```

Comment:
It looks like these are booleans, so maybe a function to get and cast them as such?